packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/adjust/adjust.component.js

```diff
@@ -0,0 +1,19 @@
+import controller from './adjust.controller';
+import template from './adjust.template.html';
+
+const component = {
+  bindings: {
+    autoscaling: '<',
+    goBack: '<',
+    kubeId: '<',
+    nodePool: '<',
+    nodePoolId: '<',
+    nodePoolName: '<',
+    projectId: '<',
+    sendKubeTrack: '<',
```

Comment:
```suggestion
    sendKubeTracker: '<',
```

Reply:
Thanks for suggestion.
This function was developed in another feature.